src/app/design/guidelines/content-containers/index.html

```diff
@@ -131,13 +131,31 @@ <h3 class="sky-subsection-heading">
       </p>
     </li>
 	</ul>
+	
+	<h3 class="sky-subsection-heading">
+    <a routerLink="/components/flyout">Flyout</a>
+  </h3>
+	
+	<p>Flyouts are non-modal containers that slide out from the side of a page and lay over page content to display large amounts of supplementary information related to user tasks. They allow users to access information without navigating to another page, and they can contain lists, record views, graphs, or bespoke information displays. Flyouts are appropriate in these scenarios:</p>
```

Comment:
What are "bespoke information displays"? I don't know what that means, but I left it in place because I don't know how to edit it.